popde/density_estimate.py

```diff
@@ -164,16 +164,24 @@ def calc_ndim_bandwidth(self, method='oned_isj'):
         Assumes a direct (plug-in or comparable) method rather than optimization
         """
         nd_bws = np.zeros(self.ndim)
-        
         # 1-d Botev et al. ("Improved Sheather-Jones") algorithm from KDEpy
-        if method == 'oned_isj':
-            from KDEpy.bw_selection import improved_sheather_jones as isj
+        #if method == 'oned_isj':
+
+        from KDEpy.bw_selection import improved_sheather_jones as isj, scotts_rule as scott, silvermans_rule as silverman
+         #dict for mapping method names to functions
```

Comment:
comment is not needed, we can see that it's a dict. 
But since 'methods' is only used once, we don't actually need to define it. 
Ie you could just do 
```
{'oned_isj': isj,
 'scott': scott,
 'silverman': silverman}.get(method)
 ```